packages/react-native-otlp/android/src/main/java/io/embrace/rnembraceotlp/RNEmbraceOTLPModule.kt

```diff
@@ -1,50 +1,158 @@
 package io.embrace.rnembraceotlp
 
-import android.util.Log
+import java.time.Duration
+import java.util.logging.Logger
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+import kotlin.time.toJavaDuration
+
 import com.facebook.react.bridge.Promise
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.bridge.ReactContextBaseJavaModule
 import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableArray
 import com.facebook.react.bridge.ReadableMap
 
 import io.embrace.android.embracesdk.Embrace
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter
+import io.opentelemetry.exporter.otlp.http.logs.OtlpHttpLogRecordExporter
+
+data class HeaderConfig (
+    val key: String,
+    val token: String
```

Comment:
again minor and sorry for not commenting on the previous PR, can these be used for any http header? Should it be `value` instead since `token` is specific to an auth header?

Reply:
yeah, I was also thinking on this. I will revisit this feedback at the end of the development process so I can be focused on testing again both sides of the implementation. Will add it as a follow up ticket

Reply:
https://www.notion.so/embraceio/Revisit-Android-iOS-Native-modules-for-improving-consistency-before-release-if-needed-1377e3c9985280698033c8dafa7f70a7?pvs=4